src/data_access/InMemoryGameDataAccessObject.java

```diff
@@ -0,0 +1,29 @@
+
+package data_access;
+
+import entity.CommonGame;
+import entity.Game;
+import use_case.create_game.CreateGameDataAccessInterface;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class InMemoryGameDataAccessObject implements CreateGameDataAccessInterface {
```

Comment:
Leaving my two cents here for documentation purposes, just in case anyone has confusion on how to deal with this situation.

In regards to addressing merge conflicts, it's common practice for the author of the PR (Nikita in this case) to do the following whenever the main branch changes (or atleast whenever it changes significantly):
- Pull the changes from the remote version of main to their local copy of main
- Locally try to merge main in to their feature branch
- If there are conflicts, resolve them. 
    - In this particular case, like Ethan hinted at, the main branch will only show the interfaces from the previous use cases being implemented while the feature branch will only show the new interface being implemented. To resolve, Nikita can just add his interface to the end of the "implements" chain on the class.
- When done resolving, git add all the changed files and make a new commit.
- Finally, push the feature branch with the merged changes back to the remote version. Done.

In my experience, frequently syncing feature branches (especially long running ones) with main will reduce the likelihood of having headaches at the end from dealing with large merge conflicts. Not saying that's the case here, just giving my opinion in general 🙂

Reply:
I just pushed my local merge into this PR branch. Preliminarily, everything seems to be working correct. I will also update the `Main.java` starter code.

Reply:
Resolving this now. I think it's better if we open a new change request if any new problems arise.

Also, thank you Rayce for the very detailed explanation!